docs/community/content/contribute/committer.cn.md

```diff
@@ -22,17 +22,49 @@ ShardingSphere社区遵循[Apache Community’s process](http://community.apache
 
 ## 日常工作
 
-1. Committer需要每天查看社区待处理的Pull Request和Issue列表，负责问题的处理：
+1. 每周负责轮值的 Committer 需要每天查看社区待处理的Pull Request 和 Issue 列表，负责问题的处理：
 
- - 包括标记issue，回复issue，关闭issue等；
- - 将issue分配至熟悉该模块的贡献者；
+ - 包括标记 issue，回复 issue，关闭 issue 等；
+ - 将 issue 分配至熟悉该模块的贡献者；
+ - 轮值周期为一周。
+1. Committer 需要每天查看社区待处理的 Pull Request 和 Issue 列表，负责问题的处理：
 
-2. Assignee在被分配issue后，需要进行如下判断：
+ - 包括标记 issue，回复 issue，关闭 issue 等；
+ - 将 issue 分配至熟悉该模块的贡献者；
 
- - 判断是否是长期issue，如是，则标记为pending；
- - 判断issue类型，如：bug，enhancement，discussion等；
- - 标记Milestone。
+2. Assignee 在被分配 issue 后，需要进行如下判断：
 
-3. Committer提交的PR，需要根据PR类型和当前发布的周期标注Label和Milestone。
+ - 判断是否是长期 issue，如是，则标记为 pending；
+ - 判断 issue 类型，如：bug，enhancement，discussion 等；
+ - 标记 Milestone。
 
-4. Committer review PR时，可以进行squash and merge to master的操作, 如果有问题可以加上change request或者@相关人员协助处理。
+3. Committer 提交的 PR，需要根据PR类型和当前发布的周期标注 Label 和 Milestone。
+
+4. Committer review PR 时，可以进行 squash and merge to master 的操作, 如果有问题可以加上 change request 或者@相关人员协助处理。
+
+### 轮值顺序表
```

Comment:
This part has been deleted in the previous version

Reply:
It is written two times, should I remove that line?

Reply:
@Swastyy please refer to this pr https://github.com/apache/shardingsphere/pull/15838/files